### DISCUSSION TITLE
just crashes - src/tcmalloc.cc:283] Attempt to free invalid pointer 0x58629be579d0

### DISCUSSION THREAD

**Discussion by mohataj** [TheLastBen/fast-stable-diffusion#2346] [Q&A] [answered]
part 1-6 successful, start stable diffusion just crashes with src/tcmalloc.cc:283] Attempt to free invalid pointer 0x58629be579d0

all setup like shown here
https://www.youtube.com/watch?v=R7GXN1kLyUk

what can i do?

**Comment by rodwd** [comment]
Same problem.

**Reply by ClaraSanders** [reply]
Here we go again. I was able to use it until today. I got this when I run the last cell: src/tcmalloc.cc:283] Attempt to free invalid pointer 0x57196510b9d0 
I already reinstalled the sd folder and I am a paid user with still computing units to use.

**Reply by rodwd** [reply]
Any success with reinstalling?

**Reply by temnoed** [reply]
i have no success after reinstall

**Reply by ClaraSanders** [reply]
Reinstalling won't solve the problem. We will have to wait.

**Reply by humansrtourists** [reply]
Same problem. No issues until today. Hope it gets fixed soon.

**Reply by ecrm77** [reply]
Google has an Ubuntu update on its system, and the Notebook should be changed to reflect that. Workaround is to go to Tools->Command Platte and activate the fallback option "Use fallback runtime version"

**Comment by ecrm77** [comment]
Google has an Ubuntu update on its system, and the Notebook should be changed to reflect that. Workaround is to go to Tools->Command Platte and activate the fallback option "Use fallback runtime version"

**Reply by rodwd** [reply]
Unfortunately do not see such a fallback option in command palette

**Reply by ecrm77** [reply]
You need to connect the runtime first, and then it will appear there.

**Reply by rodwd** [reply]
Thank you, that worked.

**Comment by TheLastBen** [comment]
use the latest notebook, but you need to set the runtime to High-RAM

**Comment by mohataj** [comment]
with fallback enabled it now does not find stable diffusion

Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 23, in <module>
    from modules import paths, timer, import_hook, errors, devices  # noqa: F401
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/paths.py", line 18, in <module>
    assert sd_path is not None, "Couldn't find Stable Diffusion in any of: " + str(possible_sd_paths)
AssertionError: Couldn't find Stable Diffusion in any of: ['/content/gdrive/MyDrive/sd/stablediffusion', '.', '/content/gdrive/MyDrive/sd']

but gdrive is connected and all successfully installed

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/879585/85e33892-62e7-4d0c-abb7-621657836c46)

**Reply by mohataj** [reply]
is working now after the current fix here

**Comment by TheLastBen** [comment, marked as answer]
fixed, use the latest notebook and don't use fallback

**Comment by mohataj** [comment]
Warning: Taming Transformers not found at path /content/gdrive/MyDrive/sd/stablediffusion/src/taming-transformers/taming
Warning: CodeFormer not found at path /content/gdrive/MyDrive/sd/stablediffusion/src/codeformer/inference_codeformer.py
Warning: BLIP not found at path /content/gdrive/MyDrive/sd/stablediffusion/src/blip/models/blip.py

something wrong with capitalization in google drive

**Reply by TheLastBen** [reply]
fixed, use the latest notebook
